runtime/moonbase/src/lib.rs

```diff
@@ -145,7 +145,7 @@ pub mod currency {
 /// Maximum weight per block
 pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_ref_time(WEIGHT_REF_TIME_PER_SECOND)
 	.saturating_div(2)
-	.set_proof_size(cumulus_primitives_core::relay_chain::v2::MAX_POV_SIZE as u64);
+	.set_proof_size(cumulus_primitives_core::relay_chain::MAX_POV_SIZE as u64);
```

Comment:
Just a thought... I wonder if it would be helpful to have Moonbase be more strict than our other runtimes for max proof size and other similar constraints. The benefit would be that we would catch things that are closer to their limits than anyone might otherwise realize.

Reply:
nvd reverted, we will take care of it in a followup PR